app/routes/wildcard.route.js

```diff
@@ -81,7 +82,7 @@ function route_wildcard (config, raneto) {
 
         }
 
-        var page_list = remove_image_content_directory(config, raneto.getPages(slug));
+        var pageList = remove_image_content_directory(config, raneto.getPages(slug));
```

Comment:
Here I just renamed the variable to be consistent with what we have in `home.route.js`.